apps/condo/domains/ticket/schema/TicketChange.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Generated by `createschema ticket.TicketChange 'ticket:Relationship:Ticket:CASCADE;'`
+ */
+
+const { Ticket } = require('./Ticket')
+const { OMIT_TICKET_CHANGE_TRACKABLE_FIELDS } = require('../constants')
+const { trackableFieldsFrom } = require('../../common/utils/serverSchema/changeTrackable')
+
+const { Relationship } = require('@keystonejs/fields')
+const { GQLListSchema } = require('@core/keystone/schema')
+const { versioned, uuided, tracked } = require('@core/keystone/plugins')
+const { SENDER_FIELD, DV_FIELD } = require('@condo/domains/common/schema/fields')
+const access = require('@condo/domains/ticket/access/TicketChange')
+const { generateChangeTrackableFieldsFrom } = require('@condo/domains/common/utils/serverSchema/changeTrackable')
+
+/**
+ *
+ * @type {GQLListSchema}
+ */
+const TicketChange = new GQLListSchema('TicketChange', {
+    schemaDoc: 'Incremental changes of Ticket',
+    fields: {
+        dv: DV_FIELD,
+        sender: SENDER_FIELD,
+
+        ticket: {
+            schemaDoc: 'Related ticket, whose change is logged in this entity',
+            type: Relationship,
+            ref: 'Ticket',
+            isRequired: true,
+            knexOptions: { isNotNullable: true }, // Required relationship only!
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
+        },
+        ...generateChangeTrackableFieldsFrom(trackableFieldsFrom(Ticket.schema, { except: OMIT_TICKET_CHANGE_TRACKABLE_FIELDS })),
```

Comment:
If we add a new field and forgot to add the resolver you should throw an exception! to prevent developer mistakes!

Reply:
Will be implemented in https://doma.atlassian.net/browse/SBERDOMA-391